### PR TITLE
Load Place objects from GeoJSON data

### PR DESCRIPTION
**Pull request by leonardr**
This branch adds a script that takes the output of a script like geojson-places-us and loads the data into the database as Place objects.

The input of the script looks like:
```
[metadata about a place]
[GeoJSON object]
[metadata about a place]
[GeoJSON object]
...
```

The GeoJSON objects are on separate lines so I don't have to load the JSON, extract the GeoJSON, and convert it back to to a string to pass it into the ST_GeomFromGeoJSON function.

I added the `abbreviated_name` field to the `Place` model so it would have a place to go. I haven't added aliases yet because I want to make sure we're prepared for place names in different languages, but I figure the official abbreviations for places are language-independent.

### CODE REVIEW

File: geography_loader.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        if parent_external_id:
+            parent = self.places_by_external_id[parent_external_id]
+        else:
+            parent = None
+
+        # This gives us a Geometry object.
+        geometry = func.ST_GeomFromGeoJSON(geography)
+
+        # We want real-world coordinates. Convert it into a Geography.
+        geography = cast(geometry, Geography)
```
Comment (leonardr) [inline geography_loader.py]: Unfortunately there is no function to convert GeoJSON into a Geography object--we have to convert it to a Geometry and then cast it.

File: scripts.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import os
+import sys
+
+from geography_loader import GeographyLoader
+from model import (
+    production_session
+)
+from config import Configuration
+
+class Script(object):
```
Comment (leonardr) [inline scripts.py]: This is copied and slightly modified from core.

File: geography_loader.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        metadata = json.loads(metadata)
+        external_id = metadata['id']
+        type = metadata['type']
+        parent_external_id = metadata['parent_id']
+        name = metadata['name']
+        aliases = metadata.get('aliases', [])
+        abbreviated_name = metadata.get('abbreviated_name', None)
+
+        if parent_external_id:
+            parent = self.places_by_external_id[parent_external_id]
```
Comment (aslagle) [inline geography_loader.py]: Do the parents always come before their children in the input?

Comment (leonardr) [inline geography_loader.py]: Yes, I do the US first, then all the states, and everything else (counties, cities, ZIP) is underneath the states.

**Pull request merged**